### ISSUE TITLE
eRAmanga: list User Requests

### ISSUE THREAD

**Issue by nathcast** [Rothamsted-Ecoinformatics/eRAcuration#40]
for User requests: 
we want to list and be able to export. 

Livewire component to list. 
model: userquest
controller : I made it a resource in case.. 
view: simple component to view the table.

**Comment by nathcast** [comment]
Ready to review: ut us on odyssey (but taking time to see the changes. 

Please see if more features are needed here. 
Do we want to edit fields? do we want to have a field to signify that the information has been entered in the other database.

**Comment by mglendining** [comment]
Maybe add the option to sort by date?

**Comment by mglendining** [comment]
Is it possible to make the column 'data requested' wider, as it can contain a lot of information.

**Comment by nathcast** [comment]
Hi: I have made the table sortable by date, and also found the function for the width which was not officially documented! 

A little note, the orange question mark moves the page to the bottom where there is help. (on all the pages) so for the urequest, I have listeed there the codes for the referal that you often have a problem with,.
It is possible that I change the original for so that what is recorded is closer to what you have in your database:: and that will be even easier for you.
